modules/couchbase/testdata/basicstat.json

```diff
@@ -0,0 +1,54 @@
+[
```

Comment:
couchbase version?


usually we use `testdata/vX.Y.Z/...`

see https://github.com/netdata/go.d.plugin/tree/master/modules/elasticsearch/testdata/